eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - ${{ if or(eq(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
         - (Ubuntu.2204.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8
       - ${{ if or(ne(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Ubuntu.1804.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8
+        - (Ubuntu.2204.Arm64.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8
```

Comment:
There's no such arm64/arm32 images.

Reply:
These got removed in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1050/files

Reply:
that PR removes 18.04 images, the 22.04 one still exists

Reply:
Would you please clarify if I need to make changes here? I don't see failures in the CI but maybe these didn't get used.